roles/openshift_metrics/vars/main.yaml

```diff
@@ -18,3 +18,8 @@ hawkular_persistence: "{% if openshift.hosted.metrics.storage.kind != None %}tru
 hawkular_type: "{{ 'origin' if deployment_type == 'origin' else 'enterprise' }}"
 
 metrics_upgrade: openshift.hosted.metrics.upgrade | default(False)
+
+hawkular_cassandra_storage_types:
```

Comment:
This is not an input to the playbook (that is `hawkular_cassandra_storage_type`, without the 's').  These are just the valid values for that variable, it is used for validation (see `roles/openshift_metrics/tasks/main.yaml`).

That said, I don't know if it justifies having it in `vars`.